server/server.js

```diff
@@ -4,12 +4,14 @@ const express = require('express')
 const server = express()
 
 const starterRoute = require('./routes/starter')
+const users = require('./routes/usersRoutes')
```

Comment:
Could you please change this line to `const usersRoutes = require('./routes/users')`
Thanks!

Reply:
Thanks @emilyparkes , this is now complete